examples/classification/scripts/multiclass/seed_test_suite.py

```diff
@@ -54,9 +55,45 @@ def main(args: Namespace) -> int:
         reset=True,
     )
 
+    # Metadata Test Cases
+    stratification_logic_map = {
+        "light": lambda brightness: brightness >= 190,
+        "dark": lambda brightness: 0 <= brightness < 90,
```

Comment:
Do we want a `medium` brightness that is in between?

Reply:
I wanted to just highlight the extreme cases but I will consider it 👍